### PR TITLE
Faithless loa faith

### PR DESCRIPTION
**Pull request by ElMariuso**
- I added a new religion, made up of those who betrayed their loas to gain their powers. So Faithless but also the Drakkari of wotlk.
- In the old version, it was a faith of loa_group but I preferred to create a new religion if only for localization. If you wanted to create a new faith that denied its loas, you would still have your loas as gods.
- People within the same religion are seen as righteous. Everyone sees apostates as hostile. Loa religions see them as evil, however.
- I adapted the history accordingly.

### CODE REVIEW

File: localization/english/religion/wc_religion_l_english.yml [hunk 1]
```diff
@@ -277,6 +277,12 @@
  cult_of_torga_adherent_plural:0 "Torgi"
  cult_of_torga_desc:0 "A pantheon of Loa founded by the tortollans. Torga, a turtle Loa, is the main Loa of this pantheon. He is a wise and benevolent Loa, and his followers frequently undertake pilgrimages to him to hear his stories."
  
+ faithless:0 "Kortheki"
+ faithless_adj:0 "Kortheki"
+ faithless_adherent:0 "Kortheki"
+ faithless_adherent_plural:0 "Kortheki"
+ faithless_desc:0 "Also called the Faithless, the Kortheki are those who turned their away from their serpent loa, Sethraliss. Heretics, they gave up the peaceful path to become machines of war, using the divine essence by force to fulfill their interests and their destiny."
```
Comment (LordSilvermane) [inline localization/english/religion/wc_religion_l_english.yml:284]: Suggested some changes.
```suggestion
 faithless_desc:0 "Also called the Faithless, the Kortheki are those who turned away from their serpent loa, Sethraliss. The heretics turned away from the path of peace to become machines of war, forcefully using the divine essence of Sethraliss to fulfill their interests and their destiny."
```

**Review by LordSilvermane — COMMENTED** [review]
As requested, looked over the description. Suggested a few changes.

**Comment by KingofMushan** [comment]
Proofread the description. Had some errors, here's an edited version to use if you want:

Also called the Faithless, the Kortheki are those Sethrak who turned away from their serpent loa, Sethraliss. Seen as heretics by their Zandalari neighbors, they gave up the peaceful path to become seekers of war, using force to fulfill their interests and their perceived destiny.

File: localization/english/religion/wc_religion_l_english.yml (hunk 1 above)
Comment (arithon) [reply localization/english/religion/wc_religion_l_english.yml:284]: Maybe "turned their back on their serpent loa" in the first sentence?

Comment (LordSilvermane) [reply localization/english/religion/wc_religion_l_english.yml:284]: Works for me, I had been torn between the two. Updated accordingly.

Comment (ElMariuso) [reply localization/english/religion/wc_religion_l_english.yml]: I choose the first proposition

File: common/religion/doctrines/wc_doctrines_religions.txt [hunk 2]
```diff
@@ -159,4 +159,14 @@
...
+	apostasy_hostility_doctrine = {
+		visible = no
+		parameters = {
+			hostility_same_religion = 3
+			hostility_same_family = 2
+			hostility_others = 1
+
+			hostility_override_apostasy_hostility_doctrine = 3
+		}
+	}
```
[the remark this answers is not in the document]
Comment (ElMariuso) [reply common/religion/doctrines/wc_doctrines_religions.txt]: Suggestion commit in 1ad4ce405

File: common/religion/doctrines/00_core_tenets.txt [hunk 3]
```diff
@@ -3033,6 +3033,35 @@
...
+			}
+		}
+
+		is_shown = {
+			evil_public_religion_trigger = no
+		}
+
+		parameters = {
+			apostasy_syncretism_active = yes
+			hostility_override_apostasy_hostility_doctrine = 3
```
[the remark this answers is not in the document]
Comment (ElMariuso) [reply common/religion/doctrines/00_core_tenets.txt]: add in 069951dc5

**Review by ercarp — APPROVED** [review]
Seems like a good addition to me. Also checked the localization and didn't see any issues there either. LGTM!

**Pull request merged**